locmap/docs/user.md

```diff
@@ -0,0 +1,245 @@
+#User Resource
+
+##Description
+Return data pertaining to a User and their Location
+
+***
+##Requires Authentication
+* Requires valid User ID when querying API
+
+***
+
+##`POST` /user/:userid/location
+Update user location
+
+###Parameters
+- **userId** _(required)_ - Id of current user
+- **request cache** _(required)_ - Current cache of user (from authorization step)
```

Comment:
Why is this listed as a parameter? What does this actually mean? Now correct me if I'm wrong. When user authenticates, his/her data is stored in the backend's locmap cache variable. That doesn't actually mean that it is a parameter.